packages/millicast-sdk/src/Signaling.js

```diff
@@ -78,7 +78,7 @@ export default class Signaling extends EventEmitter {
    */
   async connect () {
     logger.info('Connecting to Signaling Server')
-    if (this.transactionManager && this.webSocket?.readyState === WebSocket.OPEN) {
+    if (this.transactionManager != null && this.webSocket?.readyState === WebSocket.OPEN) {
```

Comment:
Is it enough only check `null` without checking `undefined`?  
```suggestion
    if (this.transactionManager && this.webSocket?.readyState === WebSocket.OPEN) {
```